src/systems/world_exporter/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+gz_add_system(world-exporter
```

Comment:
Unless we have the intention of making this a generic world exporter, could this be renamed to `collada-world-exporter` or something similar? I think it would be good to have the word `collada` in there.

Reply:
Renamed in 8d8a7f46cb0816ecdb56ad91e789a86c324039e2 and b42e9044273b77a7b41b48e611ef80f2bbc64556